src/renderer/components/status-bar/status-bar-items.injectable.ts

```diff
@@ -12,12 +12,10 @@ const statusBarItemsInjectable = getInjectable({
   instantiate: (di) => {
     const extensions = di.inject(rendererExtensionsInjectable);
 
-    return computed(() => (
-      extensions.get()
-        .flatMap(ext => ext.statusBarItems)
-    ));
+    return computed(() =>
+      extensions.get().flatMap((ext) => ext.statusBarItems),
+    );
```

Comment:
Interesting formatting change

Reply:
Prettier did it, nothing brainy behind it.